packages/coinstac-ui/app/render/components/projects/form-project.jsx

```diff
@@ -339,7 +339,7 @@ export default class FormProject extends Component {
 }
 
 FormProject.propTypes = {
-  allowComputationRun: PropTypes.bool.isRequired,
+  // allowComputationRun: PropTypes.bool.isRequired,
```

Comment:
Several were commented out before I got in there, so I didn't want to start deleting anything without someone else taking a look first. I've deleted them now along with the `//TODO`s listed in there. Let me know if I need to put them back.